### PR TITLE
Bugfix/jr2 840 fix singleselect

### PR DESCRIPTION
**Pull request by Invizible**

### CODE REVIEW

File: src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect-component.ts [hunk 1]
```diff
@@ -21,14 +19,17 @@ export class TypeaheadSingleselectComponent implements ControlValueAccessor {
     @Input() editable = true;
     @Input() size: 'sm' | 'lg' = 'sm';
 
-    selectedItem: TypeaheadMultiselectModel;
+    selectedItems: TypeaheadMultiselectModel[] = [];
```
Comment (birom) [inline src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect-component.ts]: Do we still need this component?

File: src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect.component.html [hunk 2]
```diff
@@ -1,23 +1,8 @@
...
-           [resultTemplate]="template"
-           [ngbTypeahead]="wrappedItemLoader"
-           [resultFormatter]="formatResultItem"
-           [inputFormatter]="formatInputValue"
-           [editable]="editable"
-           [placeholder]="placeholder"
-           (change)="inputChange($event)"
-    />
-</div>
+<jr2-typeahead-multiselect [ngModel]="selectedItems"
```
Comment (birom) [inline src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect.component.html]: I think we also need a tooltip here.

File: src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect-component.ts (hunk 1 above)
Comment (Invizible) [reply src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect-component.ts]: it's like a shortcut for jr2-typeahead-multiselect [limit]="1"

File: src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect.component.html (hunk 2 above)
Comment (Invizible) [reply src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect.component.html]: Toni, said we don't need it

Comment (birom) [reply src/main/webapp/app/shared/input-components/typeahead/typeahead-singleselect/typeahead-singleselect.component.html]: Thanks for the info.

**Pull request merged**